### PR TITLE
feat(replays): Add link to session replay product docs from homepage footer

### PR DESCRIPTION
**Pull request by ryan953**
This might spark a conversation, but I've added links to the (forthcoming) Session Replay product docs from the footer on the homepage:

| Before | After |
| --- | --- |
| before  | after |

Changes:
1. `Session Replay` is added to the 1st column,
2. `Sentry-CLI` moved from column 1 into 2
3. `Integrations` moved from column 2 into 3, next to Integration Platform

This is just a suggestion!
We should add a link down here imo, but what position does it take?

Also DONT MERGE this until the product docs actually exist

### CODE REVIEW

File: src/pages/index.tsx
Comment (bruno-garcia) [inline src/pages/index.tsx:165]: We'll need the landing page before we can merge, the 404 checker will break the build

Comment (bruno-garcia) [reply src/pages/index.tsx:165]: Just read the bottom of the description 🚀

**Review by bruno-garcia — APPROVED** [review]
LGTM!

**Comment by rahulchhabria** [comment]
@ryan953 - i think @lizokm should review this PR.

**Comment by lizokm** [comment]
LGTM!

**Pull request merged**
